packages/stream_chat/lib/src/db/chat_persistence_client.dart

```diff
@@ -79,11 +79,9 @@ abstract class ChatPersistenceClient {
     PaginationParams? messagePagination,
   });
 
-  /// Get stored [Draft] message by providing channel [cid].
-  Future<Draft?> getDraftMessageByCid(String cid);
-
-  /// Get stored [Draft] message by providing parent message [id].
-  Future<Draft?> getDraftMessageByParentId(String parentId);
+  /// Get stored [Draft] message by providing channel [cid] and a optional
+  /// [parentId] for thread messages.
+  Future<Draft?> getDraftMessageByCid(String cid, {String? parentId});
```

Comment:
This is a breaking change in the package, but technically adding an abstract method without default implementation is already a breaking change in the interface. I guess we never applied semantic versioning there and just assumed people always use the same version of stream_chat and stream_chat_persistence?